storyblok/storyblok-design-system
# Chore: replace base icons

## Pull request type

Jira Link: [INT-](url)




## How to test this PR


Check the icon: https://storyblok-design-system-git-chore-replace-4262cb-storyblok-com.vercel.app/?path=/docs/basic-sbicon--documentation

You can now use all the lucide icon names too 

## What is the new behavior?



-  https://storyblok-design-system-git-chore-replace-4262cb-storyblok-com.vercel.app/?path=/docs/basic-sbicon--documentation
- 
- 

## Other information
